### ISSUE TITLE
Configurable title for clear button

### ISSUE THREAD

**Issue by igorespin** [sagalbot/vue-select#1310]
**Is your feature request related to a problem? Please describe.**
Right now the title for the clear button is "Clear Selected". I need to show the title in another language so It'll be a different text.

**Describe the solution you'd like**
Maybe a prop can solve this. So I can send any text to be shown at the clear button title.

Thanks

**Comment by kamerat** [comment]
I agree it could be nice to change this. A slot for the clear button would probably be better due to full flexibility.

You can however "hack" this together yourself now.

By using the prop [`clearable`](https://vue-select.org/api/props.html#clearable) and creating your own clear button:

https://codepen.io/BlitZZ96/pen/zYBXqQY

**Comment by igorespin** [comment]
Awesome!! Thanks for the hack...
